subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -24,8 +24,14 @@
 
 static radio_isr_fp sfp_radio_isr;
 
+static u32_t last_pkt_end_time_us;
+
 void isr_radio(void)
 {
+	/* On packet END event, increment last packet end time value. */
```

Comment:
This function is used for Radio ISR sharing amongst BLE states/roles, any role specific code should be in the functions set using radio_isr_set()

Reply:
@cvinayak I am not sure I understood your comment here. Could you elaborate a bit more?

If I understood correctly: you would like the code, here, to be moved to the ISR callbacks in the roles. However, this would mean that this code would be replicated several times. I wonder if this optimal.

Reply:
Move this inside radio_is_done() interface, and conditional compile (both memory used and code). You will avoid redundant branching instructions.